docker/docker-compose.yml

```diff
@@ -1,6 +1,4 @@
 ---
-version: '3'
```

Comment:
could you please remove the changes to the docker compose from this PR? 
If they're good changes, happy to take them, but let's not mix it into this PR.